packages/pn-pa-webapp/src/components/NewNotification/__test__/Recipient.test.tsx

```diff
@@ -92,39 +84,36 @@ const testRecipientFormRendering = async (
   const digitalForm = form.querySelector(
     `input[name="recipients[${recipientIndex}].digitalDomicile"]`
   );
+  expect(digitalForm).toBeInTheDocument();
+
   const physicalForm = within(form).queryByTestId(`physicalAddressForm${recipientIndex}`);
-  if (recipient?.showPhysicalAddress) {
-    expect(physicalForm).toBeInTheDocument();
-    const address = physicalForm?.querySelector(
-      `input[name="recipients[${recipientIndex}].address"]`
-    );
-    expect(address).toHaveValue(recipient.address);
+  expect(physicalForm).toBeInTheDocument();
+  const address = physicalForm?.querySelector(
+    `input[name="recipients[${recipientIndex}].address"]`
+  );
+
+  if (recipient) {
```

Comment:
why this if?

Reply:
Because in function `testRecipientFormRendering` recipient is optional.